invenio_communities/views/ui.py

```diff
@@ -65,6 +65,26 @@ def _can_create_community():
     return can_create
 
 
+def _has_about_page_content():
+    """Function used to check if about page has content."""
+    community = request.community
```

Comment:
See comment above about using `g` instead. Very minor: I would also call it `current_community` to make it very clear.